### ISSUE TITLE
Missing library search

### ISSUE THREAD

**Issue by billettg** [pkkid/python-plexapi#1425]
### Describe the Bug

Running the below script errors as follows:

`Error: (404) not_found; https://metadata.provider.plex.tv/library/search?query=Gladiator&limit=30&searchTypes=movies%2Ctv&includeMetadata=1 {"Error":{"error":"Not Found","message":"Missing library search","statusCode":404}}`

### Code Snippets

```python
from plexapi.myplex import MyPlexAccount

# Plex API setup
PLEX_USERNAME = '<redacted>'
PLEX_PASSWORD = '<redacted>'

account = MyPlexAccount(PLEX_USERNAME, '', PLEX_PASSWORD)

def addToWatchList(query):
    try:
        movies = account.searchDiscover(query, limit=30)
        if movies:
            movie = movies[0]
            account.addToWatchlist(movie)
            print(f"Added '{movie.title}' to the watchlist.")
        else:
            print("Movie not found.")
    except Exception as e:
        print(f"Error: {e}")

addToWatchList('Gladiator')
```

### Expected Behavior

Finds movie item.

### Additional Context

Why is the script always erroring with "Missing library search"?

I checked the documentation and source here: https://python-plexapi.readthedocs.io/en/latest/modules/myplex.html#plexapi.myplex.MyPlexAccount.searchDiscover

### Operating System and Version

Ubuntu 24.04

### Plex Media Server Version

4.131.1

### Python Version

3.10.12

### PlexAPI Version

4.13.2

**Comment by billettg** [comment]
Never mind, for some reason the fresh pip install of plexapi resulted in version 4.13.2. I upgraded to plexapi 4.15.3 using `pip install plexapi --upgrade` and it works now :)

**Comment by JonnyWong16** [comment]
Was patched in #1239.
